themes/bootstrap3/js/resultcount.js

```diff
@@ -2,19 +2,29 @@
 
 VuFind.register('resultcount', function resultCount() {
   function init() {
-    $('ul.nav-tabs [data-show-counts] a').each(function queryResultCount(){
-      var $this = $(this);
-      if ($this.attr('href') !== undefined) {
-        var queryString = $this.attr('href');
-        var source = $this.data('source');
-        $.ajax({
-          url: VuFind.path + '/AJAX/JSON?method=getResultCount',
-          dataType: 'json',
-          data: {querystring: queryString, source: source},
-          success: function appendResultCount(response){
-            $this.append(' (' + response.data.total.toLocaleString() + ')');
+    document.querySelectorAll('ul.nav-tabs [data-show-counts] a').forEach((tab) => {
+      function loadCount(url) {
+        let source = tab.dataset.source;
+        let xhttp = new XMLHttpRequest();
```

Comment:
Could this be simplified using the fetch api?

Reply:
Done